documentation/prerequisites.md

```diff
@@ -7,13 +7,15 @@ Following is the list of prerequisites for running Kubernetes with vSphere Cloud
 * We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
 * We support Photon, Ubuntu, CoreOs and Centos. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3
 * vSphere setup to deploy the Kubernetes cluster.
+   - For kubernetes version 1.9.x and above: vSphere Cloud Provider supports kubernetes cluster spnning across multiple vCenters.
+   - For kubernetes version 1.8.x and below: vSphere Cloud Provider supports kubernetes cluster deployed only in one vCenter.
 * vSphere shared storage. It can be any one of VSAN, sharedVMFS, sharedNFS.
    - Shared storage allows all the nodes in the Kubernetes cluster to access the storage blocks.
+   - With kuberneters version 1.9.x if user wants to use multiple vCeneters then vSAN storage can not be used. vSAN is limited to one cluster in one vCenter deployement.
 * vCenter user with required set of privileges.
    - To know privileges required to install Kubernetes Cluster using Kubernetes-Anywhere, please check this [link](https://github.com/Kubernetes/Kubernetes-anywhere/blob/master/phase1/vsphere/README.md#prerequisites)
-   - If you are not installing Kubernetes cluster using Kubernetes-Anywhere and just enabling vSphere Cloud Provider on Kubernetes cluster already deployed on vSphere, Please refer this [link](https://kubernetes.io/docs/getting-started-guides/vsphere/#enable-vsphere-cloud-provider) to know about privileges required for Cloud Provider.
 * VMware Tools needs to be installed on the guest operating system on each Node VM. Please refer this [link](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.html.hostclient.doc/GUID-ED3ECA21-5763-4919-8947-A819A17980FB.html) for instruction on installing VMware tools.
```

Comment:
Let's just add a bullet point.
VMware tools - [version] 
No need to include instructions on how to install VMware tools.

Reply:
Let's remove this entire section as it describes the configuration and hence belongs in deployment section and NOT in prerequisites.

"vCenter user with required set of privileges.
    - To know privileges required to install Kubernetes Cluster using Kubernetes-Anywhere, please check this [link](https://github.com/Kubernetes/Kubernetes-anywhere/blob/master/phase1/vsphere/README.md#prerequisites)"

Reply:
Before enabling VCP, user should have VMware tools installed, priviledges  correctly set. So I do see a value of these steps here.

Reply:
I agree that we should retain these instructions then.